programs/merkle-tree/src/lib.rs

```diff
@@ -27,66 +27,17 @@ pub mod utils;
 pub mod config_accounts;
 pub use config_accounts::*;
 
-use crate::{
-    errors::ErrorCode,
-    transaction_merkle_tree::state::TransactionMerkleTree,
-    utils::{
-        accounts::deserialize_and_update_old_merkle_tree,
-        config::{self, MERKLE_TREE_HEIGHT, ZERO_BYTES_MERKLE_TREE_18},
-        constants::{EVENT_MERKLE_TREE_SEED, TRANSACTION_MERKLE_TREE_SEED},
-    },
-};
+use crate::utils::config::{self};
 
+#[cfg(feature = "atomic-transactions")]
```

Comment:
It's not possible to just hide one function behind a feature flag?
I thought the only function that should be impacted by this Pr is insert_two_leaves.